src/Document.php

```diff
@@ -311,4 +311,22 @@ public function registerXpathPhpFunctions($xpathPhpFunctions = true)
     {
         $this->xpathPhpFunctions = $xpathPhpFunctions;
     }
+
+    /**
+     * Disable the ability to load external XML entities based on libxml version
+     *
+     * If we are using libxml < 2.9, unsafe XML entity loading must be
+     * disabled with a flag.
+     *
+     * If we are using libxml >= 2.9, XML entity loading is disabled by default.
+     *
+     * @return bool
+     */
+    public static function disableEntityLoader($flag = true)
```

Comment:
Does this really need to be public? It's internal to the class, and could likely be safely `private`.

Reply:
It is used in another class, but I have no problem to copy the code and make the method private instead.

Reply:
https://github.com/laminas/laminas-dom/pull/8/files#diff-c6c965c2de69a9242017ed48a6aa934bb2fdaf8dca254eccb75efab6fd5b0c3eR235-R264

Reply:
I'm sorry I forgot this: so overall, what should I do? copy-paste it and make the two resulting methods `private`?

Reply:
> copy-paste it and make the two resulting methods `private`?

Yes, proceed like this.

Reply:
Done